### PR TITLE
[s] Adds a rare tips file to the tip of the day

### PR DESCRIPTION
**Pull request by Incoming5643**
Sometimes you don't want to be helpful.

Stress relief pull. The bastard descendant of #11542

### CODE REVIEW

File: config/sillytips.txt [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Cleanbot.
+The wizard is supposed to be extremely strong in one on one combat, stop getting mad about it.
+Sometimes a round will just be a bust. C'est la vie.
+This is a game that is constantly being developed for. Expect things to be added, removed, fixed, and broken on a daily basis.
+It's fun to try and predict the round type from the tip of the round message.
+The quartermaster is not a head of staff and will never be one.
+The bird remembers.
+Your sprite represents your hitbox, so that afro makes you easier to kill. The sacrifices we make for style.
+Spacemen can't move diagonally but most animals can. Blame the slow decline of the numpad.
+Sometimes admins will just do stuff. Roll with it.
```
Comment (Cheridan) [inline config/sillytips.txt]: NOOOOO NEVER

STOP SPAWNING XENOMETEORZOMBIEDRAGONS

File: code/controllers/subsystem/ticker.dm [hunk 2]
```diff
@@ -452,9 +452,12 @@ var/datum/subsystem/ticker/ticker
 
 /datum/subsystem/ticker/proc/send_random_tip()
 	var/list/randomtips = file2list("config/tips.txt")
-	if(randomtips.len)
+	var/list/memetips = file2list("config/sillytips.txt")
+	if(randomtips.len && prob(95))
 		world << "<font color='purple'><b>Tip of the round: </b>[html_encode(pick(randomtips))]</font>"
-
+	else if(memetips.len)
+		world << "<font color='purple'><b>Tip of the round: </b>[html_encode(pick(memetips))]</font>"
```
Comment (optimumtact) [inline code/controllers/subsystem/ticker.dm]: remove the string duplicattion
i.e pick value and store in temp var then do the string output once at the end of the proc

:+1:

Comment (Jalleo) [inline code/controllers/subsystem/ticker.dm]: I agree

**Comment by RandomMarine** [comment]
As a changeling, don't actually try to absorb the AI.

Carbon dioxide is a colorless and odorless gas that is produced when people breathe out or when certain fuels are burned. It kills you if you breathe too much of it.

A honk a day keeps the redshirt away.

Scooters have mysterious anti-plastic flap powers.

Don't open it.

File: config/sillytips.txt [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+The remake will never come out.
+Plenty of things that aren't traditionally considered weapons can still be used to slowly brutalize someone to death, get creative!
+DEATH IS IMMINENT!
+This game is older than some of the people playing it.
```
Comment (Supermichael777) [inline config/sillytips.txt]: should be most

File: code/controllers/subsystem/ticker.dm (hunk 2 above)
Comment (Supermichael777) [inline code/controllers/subsystem/ticker.dm]: but that would add lines

**Comment by Shadowlight213** [comment]
:+1: most of these tips are actually really good things that people should follow

**Comment by PKPenguin321** [comment]
:+1: but also feature frozen?
i like this a lot though

File: config/sillytips.txt (hunk 3 above)
Comment (Iamgoofball) [inline config/sillytips.txt]: :+1:

**Comment by PKPenguin321** [comment]
you should implement some of these: https://tgstation13.org/phpBB/viewtopic.php?f=55&t=6797&p=181759

**Comment by Incoming5643** [comment]
Wasn't expecting the fast turn around, but ok

**Comment by optimumtact** [comment]
24 hours was up  and you had plenty of support.

**Pull request merged**